packages/babel-standalone/test/babel.js

```diff
@@ -110,6 +110,61 @@
       ).toThrow(/Invalid plugin specified in Babel options: "lolfail"/);
     });
 
+    describe("env preset", () => {
```

Comment:
The [REPL Build](https://babeljs.io/repl/build/13293) is not working. I will investigate it today.